src/incubator-avatar.html

```diff
@@ -116,7 +152,9 @@
         }
 
         _setTitle(name) {
-          this.title = name;
+          if(!this._displayTooltip()) {
```

Comment:
I think you should check the `this.noTooltip` variable here instead

Reply:
The title attribute should not be set if we are going to show the tooltip.